src/System.Linq.Expressions/src/System/Linq/Expressions/DefaultExpression.cs

```diff
@@ -49,6 +49,7 @@ protected internal override Expression Accept(ExpressionVisitor visitor)
 
     public partial class Expression
     {
+        private static readonly DefaultExpression _empty = new DefaultExpression(typeof(void));
```

Comment:
Is `s_` the general convention? In any case, there's certainly a few using it in this assembly, so I'll do that in a minute.